### DISCUSSION TITLE
n00b question

### DISCUSSION THREAD

**Discussion by blake41** [mmistakes/minimal-mistakes#2884] [General]
I figured out how to add an image to the "header" of a post, but can't figure out how to do it for the frontpage of my blog.  Anyone know where the config is for this?  I've searched everywhere and can't find it.  Apologies if this is a stupid question but I'm stumped

**Comment by iBug** [comment]
Check out [`home.md`](https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_pages/home.md) and see if you're missing anything.

**Reply by blake41** [reply]
Thanks!  Finally figured out why your advice wasn't working.  I had an index.html file which was overriding this page once I added it.
